tools/alpine/Dockerfile

```diff
@@ -9,8 +9,8 @@ COPY packages* /tmp/
 
 # mirror packages
 RUN cat /tmp/packages.$(uname -m) >> /tmp/packages && \
```

Comment:
FYI, this was intentionally left as `uname -m` to hide the apk-specific details; hence the packages files in this folder would still be named after `uname -m`. Since these relate to apk packages, perhaps this should also refer to the apk architecture.